packages/video_player/video_player_avfoundation/ios/Classes/AVAssetTrackUtils.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <AVFoundation/AVFoundation.h>
+
+/**
+ * Note: https://stackoverflow.com/questions/64161544
+ * `AVAssetTrack.preferredTransform` can have wrong `tx` and `ty`
+ * on iOS 14 and above. This function provides a standardized transform
```

Comment:
oh just double check - have you tested both iOS 14+ and earlier OS version?

Reply:
Sorry I can only and have tested this on iOS14+...

Reply:
let me try it on earlier OS. could you please hold this PR for now?

Reply:
> let me try it on earlier OS. could you please hold this PR for now?

Sure. Thanks!

Reply:
Tested on iOS 13.7 and it worked. I am curious where is this "iOS 14 and above" coming from?

Reply:
this one: "used to be broken and now is working"

Reply:
let me try iOS 12 too

Reply:
it shows a blank video on iOS 12.4. Same result with or without this fix. 

![ios 12 with the change](https://user-images.githubusercontent.com/41930132/163040358-d2dfb196-3154-42b3-9133-49160c0964d4.png)

Reply:
> it shows a blank video on iOS 12.4. Same result with or without this fix.

Maybe we can file another issue to track this?

Reply:
sounds good if you think it's unrelated.